packages/auth0_fastapi/src/auth0_fastapi/server/routes.py

```diff
@@ -34,8 +35,11 @@ async def login(request: Request, response: Response, auth_client: AuthClient =
             """
 
             return_to: Optional[str] = request.query_params.get("returnTo")
+            authorization_params = {k: v for k, v in request.query_params.items() if k not in [
+                "returnTo"]}
             auth_url = await auth_client.start_login(
                 app_state={"returnTo": return_to} if return_to else None,
+                authorization_params=authorization_params,
```

Comment:
☝️  Relevant changes on this file, we forward all authorization params from the query params, which are then filtered downstream on the auth_client.